store/zksync/tokens.ts

```diff
@@ -35,7 +35,7 @@ export const useZkSyncTokensStore = defineStore("zkSyncTokens", () => {
       ethToken = explorerTokens.find((token) => token.address.toUpperCase() === ethL2TokenAddress.toUpperCase());
     }
 
-    if (eraNetwork.value.getTokens && (!baseToken || !ethToken)) {
+    if (eraNetwork.value.getTokens) {
```

Comment:
Why this bit is removed?

Reply:
Ah, this needs to be un-done before approval and merging. This was edited to allow internal bridging of tokens with the staging URL.